src/components/cohort/CohortCard.astro

```diff
@@ -2,33 +2,33 @@
 const cardsList = [
   {
     name: "Mastering Bitcoin by Andreas M. ",
-    img: "/website-v2/cohort/masteringBitcoin.png",
+    img: "/website-v2/cohort/masteringBitcoin.webp",
```

Comment:
This image is too low quality. Can we ask paperpsych for a better quality one and convert that to webp?